W-15501420-Include-ACB-on-Overview-CP (#269)
mulesoft/docs-munit: modules/ROOT/pages/coverage-maven-concept.adoc

You can choose to have either none, one, or all the report types added to your configuration.

> To configure, for example, the JSON report generation, you must add the following configuration to your `pom.xml` file:

Reviewer: ```suggestion
To configure, for example, the JSON report generation, add the following configuration to your `pom.xml` file:
```

Author: OK! Thanks!

----

<1> Each value represents a percentage. +

> If a percentage isn't defined, it defaults to *-1*, which indicates that no requirements are defined for said level so the build doesn't fail due to low coverage.

Reviewer: ```suggestion
If a percentage isn't defined, it defaults to *-1*, which sets no requirements for said level so the build doesn't fail due to low coverage.
```

Author: OK! Thanks!
